common/test/acceptance/pages/studio/settings_advanced.py

```diff
@@ -173,6 +173,7 @@ def expected_settings_names(self):
             'certificates_display_behavior',
             'course_image',
             'cosmetic_display_price',
+            'course_google_analytics_tag',
```

Comment:
Following the idea of splitting out this setting,
we should pull this list of attributes from elsewhere,
pulling them all in at once, instead of doing so on a case-by-case basis.

I can show you what I mean.